modules/ossm-cert-manager-installation.adoc

```diff
@@ -0,0 +1,238 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-cert-manager-installation_{context}"]
+= Installing cert-manager
+
+To install cert-manager, follow these steps:
```

Comment:
Consider expanding this to reflect https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc

Reply:
Here's a quick mashup you might be able to use. Please evaluate it critically before using it.
```
You can install `cert-manager` to manage the lifecycle of TLS certificates; ensuring they are valid and up-to-date. For Istio, you can also install `istio-csr`, a certificate authority (CA) server that handles certificate signing requests (CSR) from Istio proxies.
```

Reply:
Jira issue created to address this post 2.4 release. Dev input is needed.